### PR TITLE
feh: add Spanish translation

### PR DESCRIPTION
**Pull request by alexmailo**
- [x] The page(s) are in the correct platform directories: `common`.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.es/common/feh.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+
+# feh
```
Comment (marchersimon) [inline pages.es/common/feh.md:1]:
```suggestion
# feh
```

File: pages.es/common/feh.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+> Visor de imagenes.
+> Para mas informacion: <https://feh.finalrewind.org>.
+
+- Para imagenes locales o de URL:
+
+`feh {{ruta/a/imagen}}`
+
+- Para ver imagenes recursivamente :
```
Comment (marchersimon) [inline pages.es/common/feh.md:10]:
```suggestion
- Para ver imagenes recursivamente:
```

File: pages.es/common/feh.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Para cerrar despues de una imagen:
+
+`feh --cycle-once {{ruta/a/imagen}}`
+
+- Agrega una demora a la presentacion:
+
+`feh --slideshow-delay {{secundos}} {{ruta/a/imagen}}`
+
+- Cambia tu fondo de pantalla (centrado, llenar, maximizado, ampliado o amontonado ):
```
Comment (marchersimon) [inline pages.es/common/feh.md:26]:
```suggestion
- Cambia tu fondo de pantalla (centrado, llenar, maximizado, ampliado o amontonado):
```

**Review by marchersimon — COMMENTED** [review]
Thanks for your contribution. I just noticed some minor errors.

**Comment by marchersimon** [comment]
And there's no need to force-push @alexmailo. We squash all commits anyways when we merge.

**Comment by CleanMachine1** [comment]
Let @navarroaxel merge since he can guarantee the quality of this page and find any errors (if there are any)

File: pages.es/common/feh.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Para ver imagenes recursivamente:
+
+`feh --recursive {{ruta/al/directorio}}`
+
+- Para ver imagenes sin bordes:
+
+`feh --borderless {{ruta/a/imagen}}`
+
+- Para cerrar despues de una imagen:
```
Comment (navarroaxel) [inline pages.es/common/feh.md:18]:
```suggestion
- Cierra después de la última imagen:
```

File: pages.es/common/feh.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+`feh --cycle-once {{ruta/a/imagen}}`
+
+- Agrega una demora a la presentacion:
```
Comment (navarroaxel) [inline pages.es/common/feh.md:22]: Perhaps this better
```suggestion
- Agrega una demora al ciclo de la presentacion:
```

File: pages.es/common/feh.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+- Para imagenes locales o de URL:
+
+`feh {{ruta/a/imagen}}`
+
+- Para ver imagenes recursivamente:
+
+`feh --recursive {{ruta/al/directorio}}`
+
+- Para ver imagenes sin bordes:
```
Comment (navarroaxel) [inline pages.es/common/feh.md:14]:
```suggestion
- Muestra imágenes sin bordes:
```

File: pages.es/common/feh.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+# feh
+
+> Visor de imagenes.
+> Para mas informacion: <https://feh.finalrewind.org>.
+
+- Para imagenes locales o de URL:
+
+`feh {{ruta/a/imagen}}`
+
+- Para ver imagenes recursivamente:
```
Comment (navarroaxel) [inline pages.es/common/feh.md:10]:
```suggestion
- Muestra imágenes recursivamente:
```

File: pages.es/common/feh.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
+# feh
+
+> Visor de imagenes.
+> Para mas informacion: <https://feh.finalrewind.org>.
+
+- Para imagenes locales o de URL:
```
Comment (navarroaxel) [inline pages.es/common/feh.md:6]:
```suggestion
- Muestra imágenes localmente o usando una URL:
```

File: pages.es/common/feh.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+- Agrega una demora a la presentacion:
+
+`feh --slideshow-delay {{secundos}} {{ruta/a/imagen}}`
+
+- Cambia tu fondo de pantalla (centrado, llenar, maximizado, ampliado o amontonado):
+
+`feh --bg-{{center|fill|max|scale|tile}} {{ruta/a/imagen}}`
+
+- Crea un montage de todas las imagenes en un directorio. Produce una nueva imagen.
```
Comment (navarroaxel) [inline pages.es/common/feh.md:30]:
```suggestion
- Crea un montage de todas las imágenes en un directorio. Produce una nueva imagen.
```

File: pages.es/common/feh.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+- Para cerrar despues de una imagen:
+
+`feh --cycle-once {{ruta/a/imagen}}`
+
+- Agrega una demora a la presentacion:
+
+`feh --slideshow-delay {{secundos}} {{ruta/a/imagen}}`
+
+- Cambia tu fondo de pantalla (centrado, llenar, maximizado, ampliado o amontonado):
```
Comment (navarroaxel) [inline pages.es/common/feh.md:26]:
```suggestion
- Cambia el fondo de pantalla (centrado, llenar, maximizado, ampliado o amontonado):
```

File: pages.es/common/feh.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
+# feh
+
+> Visor de imagenes.
+> Para mas informacion: <https://feh.finalrewind.org>.
```
Comment (navarroaxel) [inline pages.es/common/feh.md:4]:
```suggestion
> Más información: <https://feh.finalrewind.org>.
```

File: pages.es/common/feh.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
+# feh
+
+> Visor de imagenes.
```
Comment (navarroaxel) [inline pages.es/common/feh.md:3]:
```suggestion
> Utilidad ligera de visualización de imágenes.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @alexmailo, welcome and thanks for your submission. I don't like the terms used in the English page, but I made some suggestions to the translation.

File: pages.es/common/feh.md [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+
+- Muestra imágenes sin bordes:
+
+`feh --borderless {{ruta/a/imagen}}`
+
+- Cierra después de la última imagen:
+
+`feh --cycle-once {{ruta/a/imagen}}`
+
+- Agrega una demora al ciclo de la presentacion:
```
Comment (navarroaxel) [inline pages.es/common/feh.md:22]: I forgot this
```suggestion
- Agrega una demora al ciclo de la presentación:
```

File: pages.es/common/feh.md [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+
+`feh --slideshow-delay {{secundos}} {{ruta/a/imagen}}`
+
+- Cambia el fondo de pantalla (centrado, llenar, maximizado, ampliado o amontonado):
+
+`feh --bg-{{center|fill|max|scale|tile}} {{ruta/a/imagen}}`
+
+- Crea un montage de todas las imágenes en un directorio. Produce una nueva imagen.
```
Comment (navarroaxel) [inline pages.es/common/feh.md:30]:
```suggestion
- Crea un montage de todas las imágenes en un directorio. Produce una nueva imagen:
```

**Review by navarroaxel — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**